packages/web/src/Space.tsx

```diff
@@ -133,7 +134,7 @@ export function Space(props: { match: match<{ space: string }> }) {
         </PatternFlyPopup>
       )}
 
-      <PageSection variant={PageSectionVariants.light}>
+      <div className="pf-c-page__main-nav">
```

Comment:
Will there be a React component for this div?

Reply:
I've added an issue for it. https://github.com/patternfly/patternfly-react/issues/1742